app/views/decidim/shared/_floating_help.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="floating-helper-container" data-help-id="floating-help-<%= id %>">
```

Comment:
Why we add this file?

Reply:
To modify the default behaviour (open/closed), but maybe we can do it in other way.

Reply:
I deleted the view as discussed.